rero_ils/modules/acq_order_lines/dumpers.py

```diff
@@ -56,13 +56,19 @@ def dump(self, record, data):
         }
         data['account'] = {k: v for k, v in data['account'].items() if v}
 
-        # Add document informations: pid, formatted title and ISBN identifiers.
-        # (remove None values from document metadata)
+        # Add document information's: pid, formatted title and ISBN
+        # identifiers (remove None values from document metadata)
         document = record.document
+        identifiers = document.get_identifiers(
+            filters=[IdentifierType.ISBN],
+            with_alternatives=True
+        )
+        identifiers = [identifier.normalize() for identifier in identifiers]
```

Comment:
Could you be move `normalize()` in `get_identifiers` or identifier class  ? it will save a loop.
see another dumpers too.